contract_manager/package.json

```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "build": "tsc",
     "shell": "ts-node ./src/shell.ts",
-    "lint": "eslint src/"
+    "lint": "eslint src/ scripts/",
+    "format": "prettier --write \"src/**/*.ts\" \"scripts/**/*.ts\""
```

Comment:
doesn't pre-commit cover the format part? or maybe it misses the scripts :?

Reply:
it does, but I saw it in other packages and thought maybe this is helpful for better IDE integration, etc...